homeassistant/components/hive.py

```diff
@@ -0,0 +1,707 @@
+"""Hive Integration - Platform."""
+import logging
+from datetime import datetime
+from homeassistant.helpers.discovery import load_platform

```

Comment:
Import must be ordered according PEP8.

Reply:
Imports on all files re-ordered using the recommended isort